src/js/components/FragmentGannt/fragmentGannt.jsx

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (C) 2020 Knot.x Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useEffect, useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import PropTypes from 'prop-types';
+import { constructFragmentsTimeline } from '../../helpers/nodes/nodesHelper';
+import { drawTimeline } from '../../helpers/timeline/drawHelper';
+import { GanntContainer, Timeline } from './fragmentGannt.style';
+import { setRenderedGraph } from '../../state/actions/pageData';
+
+import 'vis-timeline/dist/vis-timeline-graph2d.min.css';
```

Comment:
@SzajekSzymon - Haven't you removed `css-loader` and `style-loader` from the project in your current PR - would it cause errors here?

Reply:
Yea, I'll change it